data-prepper-core/src/main/java/org/opensearch/dataprepper/parser/PipelineParser.java

```diff
@@ -150,7 +149,7 @@ private List<Processor> newProcessor(final PluginSetting pluginSetting) {
             LOG.warn(
                     "No plugin of type Processor found for plugin setting: {}, attempting to find comparable Prepper plugin.",
                     pluginSetting.getName());
-            return newProcessor(pluginSetting, Prepper.class)
+            return newProcessor(pluginSetting, Processor.class)
                     .stream()
                     .map(prepper -> (Processor) prepper)
                     .collect(Collectors.toList());
```

Comment:
I believe this whole code block can be changed to the following now:

```
return newProcessor(pluginSetting, Processor.class);
```

Reply:
Actually, this whole catch logic can be removed. We should no longer be searching for preppers.

Reply:
Good catch! I further simplified the two `newProcessor()` methods into one on top of your comment since the second parameter `baseClass` will always be `Processor.class` now that Prepper is removed. See changes in [3b466c0](https://github.com/opensearch-project/data-prepper/pull/1707/commits/3b466c0c23a1b5c7954ab4b0b6d7a46533b8c827).